core/templates/dev/head/pages/exploration_editor/editor_tab/state_editor_responses.html

```diff
@@ -80,6 +85,7 @@
                                        is-editable="editabilityService.isEditable()"
                                        suppress-warnings="suppressDefaultAnswerGroupWarnings()"
                                        display-feedback="!isLinearWithNoFeedback(defaultOutcome)"
+                                       default-answer-group=true
```

Comment:
Missing quotes around attr value (but see comments above, better to not include this in the first place).